setup.cfg

```diff
@@ -77,11 +78,13 @@ dev =
     sphinxcontrib-openapi
     sphinxcontrib-redoc
     tox
+    types-requests
```

Comment:
For more [dependency confusion vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) avoidance, _all_ the packages in the `dev` extra should also have version numbers.